src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -203,6 +203,18 @@
 	flex-direction: column;
 }
 
+.tabs-widget-container .yellow {
+	color: yellow;
+}
+
+.tabs-widget-container .green {
+	color: green;
+}
+
+.tabs-widget-container .red {
+	color: red;
+}
```

Comment:
These will need to be brought in via the theme, we can talk about how to do this if you're not sure. Also I don't think we'll want a green color since white/black will mean everything is good.

Reply:
I no longer use those and they should be deleted